docsets/Three.js/docset.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "Three.js",
-    "version": "r77",
+    "version": "r146",
     "archive": "threejs.tgz",
```

Comment:
Please note that you should update this archive as well. I'll update it by copying `versions/r146/threejs.tgz`. If this is not ok, please let me know.

Reply:
It's correct,thank you:)